### PR TITLE
Migrated rest of the cheatcode tests to Cheatnet

### PR DESCRIPTION
**Pull request by Radinyn**

### CODE REVIEW

File: crates/cheatnet/src/cheatcodes/mock_call.rs [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 use std::collections::HashMap;
 
 impl CheatnetState {
     pub fn start_mock_call(
         &mut self,
         contract_address: ContractAddress,
-        function_name: EntryPointSelector,
-        ret_data: Vec<StarkFelt>,
+        function_name: &Felt252,
+        ret_data: &[Felt252],
```
Comment (war-in) [inline crates/cheatnet/src/cheatcodes/mock_call.rs:15]: We have to set up some guides on what we want to pass to cheatcodes

File: crates/cheatnet/tests/cheatcodes/get_class_hash.rs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    rpc::call_contract,
+};
+
+#[test]
+fn get_class_hash_simple() {
+    let mut state = create_cheatnet_state();
+
+    let contracts = get_contracts();
+    let contract_name = felt_from_short_string("HelloStarknet");
+    let class_hash = state.declare(&contract_name, &contracts).unwrap();
```
Comment (war-in) [inline crates/cheatnet/tests/cheatcodes/get_class_hash.rs:13]: I would extract the `declare` method which will take shortstring as an argument

**Review by war-in — APPROVED** [review]

File: crates/cheatnet/src/cheatcodes/mock_call.rs (hunk 1 above)
Comment (Radinyn) [reply crates/cheatnet/src/cheatcodes/mock_call.rs:15]: #506

File: crates/cheatnet/tests/cheatcodes/get_class_hash.rs (hunk 2 above)
Comment (Radinyn) [reply crates/cheatnet/tests/cheatcodes/get_class_hash.rs:13]: Should be considered in #489

**Pull request merged**